doc/dev_guide.rst

```diff
@@ -167,10 +167,13 @@ you do.
 HyperSpy uses the `pytest <http://doc.pytest.org/>`_ library for testing. The
 tests reside in the ``hyperspy.tests`` module. 
 
-First ensure pytest and its plugins are installed:
+First ensure pytest and its plugins are installed by:
 
 .. code:: bash
-
+   
+   # Either
+   pip install hyperspy[test]
+   # Or
    conda install pytest pytest-mpl freetype 
```

Comment:
freetype should be already installed and it should not be required here?

Reply:
It was specificed in the [dev_guide](https://github.com/hyperspy/hyperspy/blob/57800eab365201c81309f76fe18b18b161c1b375/doc/dev_guide.rst#plot-testing) as requiring minimum version 2.8. I don't know what it is that requires freetype, and since it isn't mentioned anywhere else I thought to include it. I'll remove it from this PR.

Reply:
Freetype may not be installed in miniconda, so I think that it's better to leave it here.

LGTM, @ericpre, if you agree with the above, could you merge this?

Reply:
freetype is a dependency of matplotlib, I don't think it is necessary to put it here and running `conda install freetype` after installing hyperspy doesn't do nothing.